pkg/crc/machine/bundle/metadata_test.go

```diff
@@ -188,24 +189,31 @@ func TestGetBundleType(t *testing.T) {
 }
 
 func TestVerifiedHash(t *testing.T) {
-	sha256sum, err := getVerifiedHash("https://developers.redhat.com/content-gateway/file/pub/openshift-v4/clients/crc/bundles/openshift/4.13.0/sha256sum.txt.sig", "crc_libvirt_4.13.0_amd64.crcbundle")
+	absPath, err := filepath.Abs(filepath.Join("testdata", "sha256sum_correct_4.13.0.txt.sig"))
+	require.NoError(t, err)
+	sha256sum, err := getVerifiedHash(fmt.Sprintf("file:///%s", absPath), "crc_libvirt_4.13.0_amd64.crcbundle")
 	require.NoError(t, err)
 	require.Equal(t, "6aad57019aaab95b670378f569b3f4a16398da0358dd1057996453a8d6d92212", sha256sum)
 
 	// sha256sum.txt is unsigned
-	_, err = getVerifiedHash("https://developers.redhat.com/content-gateway/file/pub/openshift-v4/clients/crc/bundles/openshift/4.13.0/sha256sum.txt", "crc_libvirt_4.13.0_amd64.crcbundle")
+	absPath, err = filepath.Abs(filepath.Join("testdata", "sha256sum.txt"))
+	require.NoError(t, err)
+	_, err = getVerifiedHash(fmt.Sprintf("file:///%s", absPath), "crc_libvirt_4.13.0_amd64.crcbundle")
```

Comment:
This deserves a small `testDataURI("sha256sum.txt")` helper for readability

Reply:
Done